draft-ietf-rats-corim.md

```diff
@@ -286,29 +286,29 @@ constraints must be followed when creating or validating a CoRIM map.
 The following describes each child item of this map.
 
 * `id` (index 0): A globally unique identifier to identify a CoRIM. Described
-  in {{sec-corim-id}}
+  in ({{sec-corim-id}})
 
 * `tags` (index 1):  An array of one or more CoMID or CoSWID tags.  Described
-  in {{sec-corim-tags}}
+  in ({{sec-corim-tags}})
 
 * `dependent-rims` (index 2): One or more services supplying additional,
   possibly dependent, manifests or related files. Described in
-  {{sec-corim-locator-map}}
+  ({{sec-corim-locator-map}})
 
 * `profile` (index 3): An optional profile identifier for the tags contained in
   this CoRIM.  The profile MUST be understood by the CoRIM processor.  Failure
   to recognize the profile identifier MUST result in the rejection of the
   entire CoRIM.  If missing, the profile defaults to DICE.
-  Described in {{sec-corim-profile-types}}
+  Described in ({{sec-corim-profile-types}})
 
 * `rim-validity` (index 4): Specifies the validity period of the CoRIM.
-  Described in {{sec-common-validity}}
+  Described in ({{sec-common-validity}})
 
 * `entities` (index 5): A list of entities involved in a CoRIM life-cycle.
-  Described in {{sec-corim-entity}}
+  Described in ({{sec-corim-entity}})
 
 * `$$corim-map-extension`: This CDDL socket is used to add new information
-  structures to the `corim-map`.  See {{sec-iana-corim}}.
+  structures to the `corim-map`.  See ({{sec-iana-corim}}).
```

Comment:
These look spurious.  They'd render as:

> Described in (Section 1.2.3)

which is not what we want.  Instead, they should read as:

> Described in Section 1.2.3.